docs/NewAPI.md

```diff
@@ -67,7 +67,7 @@ If you do not care about confirmation you can use producer as `Sink` and create
 
 ![Producer shape](./Producer-sink.png)
 
-To complete producer just complete it `In`.
+To complete producer just complete it `In`. <!-- something missing here?? -->
```

Comment:
Sorry, I still do not understand that sentence. Could you suggest one here and I'll inject that to the PR - or merge and correct this sentence separately?